ts/src/pro/bitfinex2.ts

```diff
@@ -661,7 +661,7 @@ export default class bitfinex2 extends bitfinex2Rest {
         if (book === undefined) {
             return;
         }
-        const depth = this.safeInteger (subscription, 'len');
+        const depth = 25; // covers the first 25 bids and asks
```

Comment:
@pcriadoperez this does not seem to be fixing it
```
 'datetime': None,
 'nonce': None,
 'symbol': None,
 'timestamp': None}
2023-04-15T13:59:52.124Z message [91247,"cs",-564698696]
Traceback (most recent call last):
  File "/Users/cjg/Git/ccxt2/ccxt/examples/py/cli.py", line 232, in <module>
    asyncio.run(main())
  File "/opt/homebrew/Cellar/python@3.10/3.10.9/Frameworks/Python.framework/Versions/3.10/lib/python3.10/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/opt/homebrew/Cellar/python@3.10/3.10.9/Frameworks/Python.framework/Versions/3.10/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/Users/cjg/Git/ccxt2/ccxt/examples/py/cli.py", line 211, in main
    result = await method(*args)
  File "/Users/cjg/Git/ccxt2/ccxt/python/ccxt/pro/bitfinex2.py", line 513, in watch_order_book
    orderbook = await self.subscribe('book', symbol, self.deep_extend(request, params))
  File "/Users/cjg/Git/ccxt2/ccxt/python/ccxt/pro/bitfinex2.py", line 62, in subscribe
    result = await self.watch(url, messageHash, self.deep_extend(request, params), messageHash, {'checksum': False})
  File "/Users/cjg/Git/ccxt2/ccxt/python/ccxt/async_support/base/ws/fast_client.py", line 27, in handler
    self.handle_message(message)
  File "/Users/cjg/Git/ccxt2/ccxt/python/ccxt/async_support/base/ws/aiohttp_client.py", line 32, in handle_message
    self.handle_text_or_binary_message(message.data)
  File "/Users/cjg/Git/ccxt2/ccxt/python/ccxt/async_support/base/ws/aiohttp_client.py", line 27, in handle_text_or_binary_message
    self.on_message_callback(self, decoded)
  File "/Users/cjg/Git/ccxt2/ccxt/python/ccxt/pro/bitfinex2.py", line 1056, in handle_message
    return method(client, message, subscription)
  File "/Users/cjg/Git/ccxt2/ccxt/python/ccxt/pro/bitfinex2.py", line 625, in handle_checksum
    payload = ':'.join(stringArray)
TypeError: sequence item 0: expected str instance, float found
bitfinex2 requires to release all resources with an explicit call to the .close() coroutine. If you are using the exchange instance with async coroutines, add `await exchange.close()` to your code into a place when you're done with the exchange and don't need the exchange instance anymore (at the end of your async coroutine).
Unclosed client session
```

Reply:
good catch, put in a fix for python and php